vnext/src/Libraries/Components/View/View.windows.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+'use strict';
+
+import type {ViewProps} from './ViewPropTypes';
+
+const React = require('react');
+import ViewNativeComponent from './ViewNativeComponent';
+const TextAncestor = require('../../Text/TextAncestor');
```

Comment:
>uire('.. [](start = 24, length = 8)

Just checking to see if the mix of `require` and `import from` is intentional.. 
Or does flow only support a subset of import semantics?

Reply:
no `warnonce` uses the default export... So curious why it's not: `import Textancestor from '../../Test/TextAncestor'`

---
In reply to: [475729169](https://github.com/microsoft/react-native-windows/pull/5818#discussion_r475729169) [](ancestors = 475729169)

Reply:
note: same applies to `view.win32.js`

---
In reply to: [475730011](https://github.com/microsoft/react-native-windows/pull/5818#discussion_r475730011) [](ancestors = 475730011,475729169)

Reply:
This file is forking the View in RN core. The mix of import/require was upstream, so we're just keeping that part as is.